change/@microsoft-teams-js-b7743f88-f4a7-4485-a6b8-2a7512bb9d83.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Updated various package versions",
+  "packageName": "@microsoft/teams-js",
+  "email": "jadahiya@microsoft.com",
+  "dependentChangeType": "none"
+}
```

Comment:
Is this changefile supposed to be in here?

Reply:
Nope this was for another PR that got jumbled while I was pushing changes to both. I've removed it now